server/index.coffee

```diff
@@ -38,17 +38,6 @@ getLocals = (extra) ->
 
 doxx.loadLunrIndex()
 
-app.get "#{config.pathPrefix}/search-results", (req, res) ->
```

Comment:
We're not using this route.

If we have results, algolia will handle the redirection.